src/main/java/com/atherys/towns/util/MathUtils.java

```diff
@@ -15,7 +15,7 @@ public static Vector2i vec3dToVec2i(Vector3d vector3d) {
     }
 
     public static double getDistanceBetweenPoints(Vector3d point1, Vector3d point2) {
```

Comment:
Now that it returns distanceSquared I would name the function that as well ;)

Reply:
Don't worry It will be lol. Thank you. Just wanted to get the logic pushed up so I could keep working from my other PC :)